sys/fuchsia/debuglog.txt

```diff
@@ -0,0 +1,27 @@
+# Copyright 2022 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+include <zircon/syscalls.h>
+
+resource zx_log[zx_handle]
+
+debuglog_create(resource_ zx_handle, options flags[debuglog_create_options], out ptr[out, zx_log])
+debuglog_write(handle zx_log, options flags[debuglog_write_options], buffer ptr[in, array[int8]], buffer_size len[buffer])
+debuglog_read(handle zx_log, options flags[debuglog_read_options], buffer ptr[out, array[int8]], buffer_size len[buffer])
+
+debuglog_create_options = ZX_LOG_FLAG_READABLE
+debuglog_write_options = 0
+debuglog_read_options = 0
+
+# unsupported on all arches (typo?)
```

Comment:
It may be related to absence of .const files.

Reply:
Right, it's literally unused. It probably returned by debuglog_read, but since we use a plain array[int8] there (which is reasonable), this type is unused and we don't need it in the descriptions.

Reply:
Removed.